tests/Http/Ownership/RequestOwnershipRequestHandlerTest.php

```diff
@@ -38,6 +39,9 @@ class RequestOwnershipRequestHandlerTest extends TestCase
     /** @var OwnershipSearchRepository&MockObject */
     private $ownerShipSearchRepository;
 
+    /** @var UserIdentityResolver */
+    private $identityResolver;
```

Comment:
This doctype is incorrect, it is a mock

Reply:
Ah yes I forgot the `&` part my bad will add it